ec_grasp_planner/data/object_param.yaml

```diff
@@ -1,48 +1,48 @@
   apple:
```

Comment:
We need to add to the readme this is generated and filled out.  @JannisW do you know what you would need to write or should we do it?

Reply:
I think you might be better suited for that. I could write something, but it wouldn't be as detailed as your explanation and I'd definitely miss something important. Shouldn't take you too long anyway, right?

Reply:
I'll do it.

Reply:
done